Library/Homebrew/utils/curl.rb

```diff
@@ -223,7 +223,7 @@ def curl_headers(*args, wanted_headers: [], **options)
         )
 
         # 22 means a non-successful HTTP status code, not a `curl` error, so we still got some headers.
```

Comment:
Maybe mention about exit code 8 here too

Reply:
Good point. I dug into this a bit more and I may need to rework the `wanted_headers` logic, so I'll address this in the next push 👍